core/services/container/Mirror.php

```diff
@@ -91,11 +101,11 @@ public function getConstructor($class_name)
 
     /**
      * @param ReflectionClass $reflection_class
-     * @return ReflectionMethod
+     * @return ReflectionMethod|null
      * @throws InvalidDataTypeException
      * @throws ReflectionException
      */
-    public function getConstructorFromReflection(ReflectionClass $reflection_class)
+    public function getConstructorFromReflection(ReflectionClass $reflection_class): ?ReflectionMethod
```

Comment:
TIL, **Nullable types** 👌

Reply:
PHP got some really nice additions in versions 7+